modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/css/_ratings_animated.scss

```diff
@@ -7,17 +7,8 @@
 			transform: scale(1.3);
 		}
 
-		// LPS-114508
-
-		> .c-inner.c-inner.c-inner.c-inner {
-			&,
-			span {
-				font-size: 1rem;
-				line-height: 1;
-			}
-		}
-
 		.on {
+			display: flex;
```

Comment:
I do not think these implementations in _ratings_animated.scss are being utilized. I still see the buttons utilize a form of .c-inner from a different .scss.

Reply:
This css applies when you click the thumbs up or down button. It changes the icon and does the animations. I think this is also used for `type="like"` but I don't remember from earlier.